crates/solidity/outputs/cargo/crate/src/generated/bindings/generated/built_ins/0.8.0.sol

```diff
@@ -0,0 +1,19 @@
+// This file is generated automatically. Do not edit by hand.
+contract $BuiltIns$ {
+  function addmod(uint x, uint y, uint k) public returns (uint);
```

Comment:
not blocking for this PR of course:
it might be a good time to think about how to add natspec doc-comments for these APIs. strings inside `definition.rs` model? another way?
where can we fetch the existing documentation from? is it available already?

Just TOL.

Reply:
Documentation for the built-ins is not made available from Slang currently. I don't know if it's readily available from Solidity itself though. For example, to figure out the exact valid versions where they became available, I had to go through their changelog version by version since I couldn't find that information anywhere else. Some of it is mentioned in the documentation, but it's very incomplete.